.gitignore

```diff
@@ -5,3 +5,4 @@ neardev/
 src/test/build/
 src/tests/build/
 src/tests/node_modules
+Cargo.lock
```

Comment:
@joshuajbouw In the future, please ensure all text files are terminated by a newline.